packages/loader/container-loader/src/container.ts

```diff
@@ -1397,7 +1413,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     private processRemoteMessage(message: ISequencedDocumentMessage): IProcessMessageResult {
-        const local = this._clientId === message.clientId;
+        const local = this.clientIdHistory.includes(message.clientId);
```

Comment:
>this.clientIdHistory.includes(message.clientId); [](start = 22, length = 48)

this changes the meaning of  local. we shouldn't change that, as this could have large implications.

Reply:
Oh I thought that was part of the mitigation >_>